conanfile.py

```diff
@@ -10,24 +10,27 @@ class OpenH264Conan(ConanFile):
     name = "openh264"
     version = "1.7.0"
     url = "https://github.com/bincrafters/conan-openh264"
+    author = "bincrafters <bincrafters@gmail.com>"
     homepage = 'http://www.openh264.org/'
     description = "Open Source H.264 Codec"
-    license = "BSD 2-Clause"
+    topics = "conan", "h264", "codec", "video", "compression",
```

Comment:
once again:

please use braces in the future `topics = (`

Reply:
ok, is this checked in bincrafters-conventions?